### PR TITLE
feat 5-timer framework

### LINKED ISSUE #5: create a timer framework for profiling at 4 seconds of interval

### PR DESCRIPTION
**Pull request by Kunalbhandari**

### CODE REVIEW

File: src/components/FunctionalConditional.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    //const [color, setColor] = useState(props.color);
+    const [num, setNum] = useState(0);
+
+    useEffect(() => {
+        setTimeout(() => {
+            setNum((num) => num + 1)
+        }, 4000)
+    }, []);
+
+    // function setdelay(){
```
Comment (rogueCoder33) [inline src/components/FunctionalConditional.js]: remove these comments

File: src/components/FunctionalConditional.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+function FunctionConditional(props){
+
+    //const [color, setColor] = useState(props.color);
+    const [num, setNum] = useState(0);
+
+    useEffect(() => {
+        setTimeout(() => {
+            setNum((num) => num + 1)
+        }, 4000)
```
Comment (rogueCoder33) [inline src/components/FunctionalConditional.js]: can we make this number driven by props/config - 4000 ?

Comment (Kunalbhandari) [reply src/components/FunctionalConditional.js]: done

File: src/components/FunctionalConditional.js (hunk 1 above)
Comment (Kunalbhandari) [reply src/components/FunctionalConditional.js]: sure, missed it

**Review by rogueCoder33 — APPROVED** [review]

**Pull request merged**